app/views/schools/show.html.erb

```diff
@@ -19,9 +19,57 @@
   <%= render 'shared/dashboard_alerts', dashboard_alerts: @dashboard_alerts, school: @school, content_field: :management_dashboard_title %>
 <% end %>
 
+<% if @suggest_estimates_for_fuel_types.present? && @suggest_estimates_for_fuel_types.any? %>
+  <%= render 'management/schools/prompt_for_estimate', school: @school, fuel_types: @suggest_estimates_for_fuel_types %>
+<% end %>
+
+<% if @prompt_for_bill %>
+  <%= render 'management/schools/prompt_for_bill', school: @school %>
+<% end %>
+
+<% if @prompt_training %>
```

Comment:
`if current_user.prompt_for_bill?`  (if moved to the user model)?
`if current_user.prompt_training? || !@show_data_enabled_features`  (if moved to the user model)?